.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,18 +12,41 @@ are the most critical to review.
 
 closes: #XXXX
 
+
+### Commit Message / Changelog entry
+
+```bash
+type: commit message
+```
+
+see the [guidelines](../CONTRIBUTING.md#commit-messages) for commit messages. (view raw markdown for examples)
```

Comment:
Maybe move this into the comment section?

Reply:
this was intended to be short version with a prompt to look at the raw markdown (it might not be obvious if you don't know where to look). 

Do you think it's not necessary to indicate that there is more details in comments?

Reply:
I was just thinking this would show on every pr description, but it is only relevant to the pr opener and thus it might make sense to make it a comment in the description so it isn't viewable by the reviewers. But I have no preference, we can always change it later

Reply:
I think it might provide some value, but like you said we can always remove it in the future if it just ends up being perceived as bloat!